src/commands/title-escrow/reject-surrendered.ts

```diff
@@ -0,0 +1,47 @@
+import { Argv } from "yargs";
+import { error, success, info } from "signale";
+import { getLogger } from "../../logger";
+import { withGasPriceOption, withNetworkAndKeyOption } from "../shared";
+import { TitleEscrowSurrenderDocumentCommand } from "./title-escrow-command.type";
+import { rejectSurrendered } from "../../implementations/title-escrow/reject-surrendered";
+import { getEtherscanAddress } from "../../utils";
+
+const { trace } = getLogger("surrender:title-escrow");
```

Comment:
```suggestion
const { trace } = getLogger("reject-surrender:title-escrow");
```

Reply:
ok renamed them to their respective file names.